InternshipBE/BL/Mapping/Profiles/CandidateProfile.cs

```diff
@@ -10,22 +9,16 @@
 
 namespace BL.Mapping.Profiles
 {
-    public class CandidateProfile : Profile
+	public class CandidateProfile : Profile
 	{
-		private readonly IGoogleConfig _googleSheetConfig;
-
-		public CandidateProfile(IGoogleConfig googleSheetConfig)
-		{
-			_googleSheetConfig = googleSheetConfig;
-		}
+		private const string cultureFormat = "dd.MM.yyyy H:mm:ss";
```

Comment:
Will be better use culture format that we set into appsettings.json instead of this const